docs/features/remote-payments.md

```diff
@@ -0,0 +1,96 @@
+# WooCommerce Remote Payment Gateway Recommendations
+
+This feature allows a remote data source to specify the currently recommended payment gateways.  The feature polls the remote JSON file to retrieve a list of recommended and visible gateways based on rulesets to be shown to the user.
+
+Plugins from this source will then walk through an installer step, followed by a connection step with the minimum required fields for setup defined by the downloaded plugin.
+
+## Enabling Remote Payment Gateway Recommendations
+
+This feature is behind a feature flag.  In order for it to run, the `remote-payment-methods` must be set to `true` in `~/config/{environment}.json`.
```

Comment:
Note that I believe this needs to be set pre-build? Either that or you need to make the change in `/includes/feature-config.php`.

Reply:
Oof, I just lost 30 minutes of my day debugging because I changed the config _after_ running npm start 😢

Reply:
Maybe we can add those config files to what webpack watches for changes

Reply:
I actually often just alter `features.php`, when in doubt. Is that bad? 😨